specification/keyvault/Security.KeyVault.Administration/tspconfig.yaml

```diff
@@ -48,8 +48,7 @@ options:
     use-model-reader-writer: false
   "@azure-tools/typespec-ts":
     package-dir: "keyvault-admin"
-    generate-metadata: true
```

Comment:
true is the default for generate-metadata

we don't want to generate tests since the package already has hand-authored tests